SPHINXsys/src/shared/particles/base_particles.cpp

```diff
@@ -299,12 +284,12 @@ namespace SPH
 			for (size_t i = 0; i != total_surface_particles; ++i) {
 				size_t particle_i = surface_particles.body_part_particles_[i];
 				Vec3d vector_value = upgradeToVector3D(variable[particle_i]);
-				output_file << std::fixed << std::setprecision(9) << vector_value[0] << " " << vector_value[1] << " " << vector_value[2] << " ";
+				output_file << std::fixed << std::setprecision(2) << vector_value[0] << " " << vector_value[1] << " " << vector_value[2] << " ";
```

Comment:
would the precision of 2 be enough for all cases?

Reply:
right now yes because it's in mm